### ISSUE TITLE
MySqlStorage.InitializeAsync syntax error occurred when using Mysql 5.5.44-0ubuntu0.12.04.1 

### ISSUE THREAD

**Issue by ajdwfnhaps** [dotnetcore/CAP#65]
```
CREATE TABLE IF NOT EXISTS `cap.published` (
  `Id` int(127) NOT NULL AUTO_INCREMENT,
  `Name` varchar(200) NOT NULL,
  `Content` longtext,
  `Retries` int(11) DEFAULT NULL,
  `Added` datetime(6) NOT NULL,
  `ExpiresAt` datetime(6) DEFAULT NULL,
  `StatusName` varchar(40) NOT NULL,
  PRIMARY KEY (`Id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
[Err] 1064 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '(6) NOT NULL,
  `ExpiresAt` datetime(6) DEFAULT NULL,
  `StatusName` varchar(4' at line 6
```

replaces datetime(6) with datetime, then working fine.

**Comment by yuleyule66** [comment]
> MySQL 5.6.4 and up expands fractional seconds support for TIME, DATETIME, and TIMESTAMP values, with up to microseconds (6 digits)  https://dev.mysql.com/doc/refman/5.6/en/datetime.html

I will adjust the syntax! 
Thanks your feeback.

**Comment by yuleyule66** [comment]
Fixed in version ` 2.1.2-preview-28879945`
